### PR TITLE
Bug 1766998, added information about the watch command

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1766998

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@thom-at-redhat Can you please review? Thank you!

**Comment by ahardin-rh** [comment]
The reporter provided review and approval in the Bugzilla.

@gpei Can you please provide QE review or direct me to the right contact? Thank you!

File: modules/update-upgrading-cli.adoc [hunk 1]
```diff
@@ -129,6 +129,12 @@ date, where the newest update is first in the list. Updates in the history have
 state `Completed` if the rollout completed and `Partial` if the update failed
 or did not complete.
 +
+You can use the `watch` command to follow the status:
++
+----
+$ watch -n 5 './oc get clusterversion -o json|jq ".items[0].status.history"
```
Comment (gpei) [inline modules/update-upgrading-cli.adoc]: Missing single quote in the end
-> `watch -n 5 './oc get clusterversion -o json|jq ".items[0].status.history"'`

Comment (ahardin-rh) [reply modules/update-upgrading-cli.adoc]: @gpei Updated. Thank you!

File: modules/update-upgrading-cli.adoc [hunk 2]
```diff
@@ -129,6 +129,12 @@ date, where the newest update is first in the list. Updates in the history have
 state `Completed` if the rollout completed and `Partial` if the update failed
 or did not complete.
 +
+You can use the `watch` command to follow the status:
```
Comment (kalexand-rh) [inline modules/update-upgrading-cli.adoc]: We aren't supposed to add new jq commands.

Comment (ahardin-rh) [reply modules/update-upgrading-cli.adoc]: Oh! I forgot. This came up recently in our team meeting. I am going to close the PR and bug then. 
@vikram-redhat FYI

Comment (ahardin-rh) [reply modules/update-upgrading-cli.adoc]: The reporter of this BZ is not sure if there's an alternative command we can use.
@sferich888 Are you able to suggest one?

Comment (sferich888) [reply modules/update-upgrading-cli.adoc]: `oc get clusterversion -o jsonpath='{range .items[0].status.history[*]}Completion Time: {.completionTime} Version: {.version} State: {.state}{"\n"}{end}' -w`

This gets ride of the `watch` command and the `jq` command. 

- Note: `-w` here will only show you state changes, from OpenShift. So we may not want to include this in the docs. 
   - That said, droping the watch command entirely may not be a bad idea.

Comment (vikram-redhat) [reply modules/update-upgrading-cli.adoc]: @gpei are you able to verify this updated command by Eric?

@ahardin-rh sorry, I missed that it was asking for a jq based command.

Comment (gpei) [reply modules/update-upgrading-cli.adoc]: @vikram-redhat It works for me

Comment (ahardin-rh) [reply modules/update-upgrading-cli.adoc]: Thank you, all! We have  ` oc get clusterversion -o json|jq ".items[0].status.history" ` already in this section of the docs, so I am going to leave that as-is and close this PR to drop mention of the watch command, then close this BZ.

**Comment by ahardin-rh** [comment]
Closing per https://github.com/openshift/openshift-docs/pull/18314#discussion_r354400777

**Pull request closed without merging**